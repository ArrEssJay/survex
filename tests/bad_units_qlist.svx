; pos=fail error=3
*fix 1 reference 0 0 0
*units position metres
*units plumb metres
*units level metres
