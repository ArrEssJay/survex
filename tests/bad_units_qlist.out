./bad_units_qlist.svx:2:8: Unknown instrument "position"
./bad_units_qlist.svx:3:8: Unknown instrument "plumb"
./bad_units_qlist.svx:4:8: Unknown instrument "level"

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 3 error(s) - no output files produced.
